### ISSUE TITLE
jest-html-reporter >> Error: Invalid character (�) in string

### ISSUE THREAD

**Issue by mohanraj-r** [salesforce/sa11y#20]
When a test fails with a11y issues and `jest-html-reporter` is being used, there is an error about an "Invalid character". Investigate and fix the error.

```
jest-html-reporter >> Error: Invalid character (�) in string: Error: expect(1 issues)toBeAccessible: expected document to have no accessibility violations but found 1 issues:

 "⭕ (image-alt) Images must have alternate text: foo:nth-child(2),img"
        🔗 Help URL: https://dequeuniversity.com/rules/axe/3.5/image-alt(0 issues)
```
